doc/sphinx/changes.rst

```diff
@@ -14,6 +14,100 @@ Version 8.14
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+Coq version 8.14 integrates many usability improvements, as well
+as an important change in the core language. The main changes include:
+
+  - The :ref:`internal representation <814CaseRepresentation>` of `match` has changed
+    to a more space-efficient and cleaner structure, allowing to fix a completeness
+    issue in presence of cumulative inductive types in the type-checker.
+    The internal representation is now closer to the user-level view of `match`,
+    where the argument context of branches and the inductive binders `in` and `as`
+    do not carry type annotations.
+
+  - A :ref:`new <814CoqNative>` `coqnative` binary performs separate native compilation
+    of libraries, starting from a `vo` file, and is supported by `coq_makefile`.
+    :ref:`Linking of native code libraries <814NativeLinking>` is now performed lazily,
+    bringing a gain of performance on some systems.
+
+  - :ref:`Improvements <814TCCanon>` of typeclasses and canonical structure
+    resolution, allowing more terms to be considered as classes or keys.
+
+  - More control of :ref:`notations <814Notations>` declarations and support
+    for primitive types in string and numeral notations.
+
+  - :ref:`Removal <814Tactics>` of deprecated tactics, notably `omega`, replaced by
+    a greatly improved `lia`, along with many bugfixes.
+
+  - New APIs for interaction with Ltac1, manipulation of inductive types and
+    a Printf module in :ref:`Ltac2 <814Ltac2>`.
+
+  - CoqIDE now integrate an :ref:`Ltac debugger <814CoqIDE>`.
+
+  - Many :ref:`changes and additions <814Stdlib>` to the standard library in the numbers,
+    vectors and lists libraries. A new signed primitive integers library `Sint63`
+    is available in addition to the unsigned `Uint63` library.
+
+  - Coq has been :ref:`split <814Dune>` into `coq-core` and `coq-stdlib` packages
+    that are now built using `dune`.
+
+See the `Changes in 8.14.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual <https://coq.github.io/doc/v8.14/refman>`_,
+`documentation of the standard library <https://coq.github.io/doc/v8.14/stdlib>`_
+and `developer documentation of the ML API <https://coq.github.io/doc/v8.14/api>`_
+are also available.
+
+Maxime Dénès, Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop and Théo Zimmermann worked on maintaining and improving the
```

Comment:
Do we usually list names in alphabetic order?

Reply:
Yes